source/extensions/network/dns_resolver/apple/apple_dns_impl.cc

```diff
@@ -228,7 +229,7 @@ void AppleDnsResolverImpl::PendingResolution::finishResolve() {
   ENVOY_LOG_EVENT(debug, "apple_dns_resolution_complete",
                   "dns resolution for {} completed with status {}", dns_name_,
                   static_cast<int>(pending_response_.status_));
-  callback_(pending_response_.status_, std::move(finalAddressList()));
+  callback_(pending_response_.status_, pending_response_.details_, std::move(finalAddressList()));
```

Comment:
We probably also want to set details here: https://github.com/envoyproxy/envoy/blob/693a94620a844f8ffde5897771bee4888dc013b1/source/extensions/network/dns_resolver/apple/apple_dns_impl.cc#L118

Reply:
yeah thankfully the compiler caught me as well